### DISCUSSION TITLE
Input autocomplete="off"

### DISCUSSION THREAD

**Discussion by IronTony** [chakra-ui/chakra-ui#4825] [Q&A]
Hi all,

I am in this situation, I have a page with a login form, I fill username and password and based on the API response I set a state to display a new form to reset the password.
Actually when I'm redirected to the reset password form, the new-password field is already filled with the password I added in the login page.
I added **_autocomplete="off/none/nope_** to the _Input_ and so on, but that field is always pre-filled.

Any help?

**Comment by chr-ge** [comment]
Have you tried `autoComplete="off"`?

**Reply by IronTony** [reply]
yes also...same result

**Reply by theboman** [reply]
did you ever figure out a solution? I am having the same issue...

**Reply by peterDuot-stationfive** [reply]
Follow... I've been experiencing this one also, inputs wrapped with `FormControl` seems to cause the autofill/autocomplete

**Reply by theboman** [reply]
should be possible to make a ref or in the case of chakra a forwardRef to grab the DOM element and remove a readonly attribute once it has focus... this is the vanilla JS version... I'll work on this today and let you know how it fairs... 
`<input type="password" name="Password" autocomplete="off" readonly 
    onfocus="this.removeAttribute('readonly');" >`

**Reply by brigix** [reply]
I had same issue, helped add to password input field "autoComplete="new-password"". Email as well became empty as I did not needed autocompletion at all (new users registration form where you have to register other users, not self). Hope it helps for someone.

**Reply by DedStriger** [reply]
"autoComplete="new-password"" still works in 2025 for chakra ui v2.8.2

**Comment by coderbahrom** [comment]
@IronTony The same problem in my app but if you have extra input if its types is password even if you did off autoComplete it causes autofill again
